draft-ietf-quic-recovery.md

```diff
@@ -683,12 +683,12 @@ client receives Handshake or 1-RTT packets prior to obtaining Handshake keys,
 it may assume some or all of the server's Initial packets were lost.
 
 To speed up handshake completion under these conditions, an endpoint MAY, for a
-limited number of occasions per each connection, send a packet containing
+limited number of times per connection, send a packet containing
 unacknowledged CRYPTO data earlier than the PTO expiry, subject to the address
 validation limits in Section 8.1 of {{QUIC-TRANSPORT}}. Doing so at most once
 for each connection is adequate to quickly recover from a single packet loss.
-Endpoints that do not cease retransmitting packets in response to
-unauthenticated data risk creating an infinite exchange of packets.
+An endpoint that always retransmits packets in response to receiving packets
+that it cannot process risks creating an infinite exchange of packets.
```

Comment:
```suggestion
An endpoint that always retransmits packets in response to receiving
unprocessable packets risks creating an infinite exchange of packets.
```

Reply:
I don't like this change.

Reply:
Yeah, I think that this changes what an endpoint cannot do right now to a property of the packets. I'm reverting this change.